crates/bevy_ui/src/flex/mod.rs

```diff
@@ -48,27 +48,25 @@ impl fmt::Debug for FlexSurface {
 impl Default for FlexSurface {
     fn default() -> Self {
         Self {
-            entity_to_stretch: Default::default(),
+            entity_to_sprawl: Default::default(),
             window_nodes: Default::default(),
-            stretch: Stretch::new(),
+            sprawl: Taffy::new(),
         }
     }
 }
 
 impl FlexSurface {
     pub fn upsert_node(&mut self, entity: Entity, style: &Style, scale_factor: f64) {
         let mut added = false;
-        let stretch = &mut self.stretch;
-        let stretch_style = convert::from_style(scale_factor, style);
-        let stretch_node = self.entity_to_stretch.entry(entity).or_insert_with(|| {
+        let sprawl = &mut self.sprawl;
```

Comment:
I'll not put suggestions on all of these, but these should be `sprawl` -> `taffy` before merging

Reply:
@Weibye thanks for the heads up! I've fixed them all I think :)